src/drt/src/db/obj/frNet.h

```diff
@@ -272,6 +274,8 @@ class frNet : public frBlockObject
   bool hasInitialRouting_{false};
   bool isFixed_{false};
 
+  // Flag to mark when a frNet has a jumper
```

Comment:
I think you misunderstood my review. This comment could have a brief explanation about what is the jumper, something like:
"Flag to mark when a frNet has a jumper, which is a special route guide used to prevent antenna violations"